README.md

```diff
@@ -8,24 +8,65 @@ ospool:
 In this lesson, we'll learn how to troubleshoot jobs that never start or fail in unexpected ways. 
 
 # Troubleshooting techniques
+We will look into 4 criterias. The criteria's are listed below
 
-## Diagnostics with condor_q
+ - [Job is held](#job-is-held)
+ - [Job completed but was unsuccessful](#job-completed-but-was-unsuccessful)
+ - [Job does not start](#Job-does-not-start)
+ - [Job is running longer than expected](#job-is-running-longer-than-expected)
 
-The `condor_q` command shows the status of the jobs and it can be used 
-to diagnose why jobs are not running. Using the `-better-analyze` flag 
-with `condor_q` can show you detailed information about why a job isn't 
-starting on a specific pool. Since OSG Connect sends jobs to many places, we also need to 
-specify a pool name with the `-pool` flag.                              
+## Job is held
+HTCondor puts your jobs on hold when something goes wrong in the process of managing your jobs. HTCondor provides a ``Hold Reason`` that explains what went wrong. To see what went wrong you can use the command `condor_q -hold`. A typical hold message may look like the following
 
-Unless you know a specific pool you would like to query, checking the `flock.opensciencegrid.org` pool is usually a good place to start.
+	$ condor_q -hold
+	ap40.uw.osg-htc.org : <128.105.68.62:9618?... @ 08/01/24 15:01:50
+ 	ID         OWNER          HELD_SINCE  HOLD_REASON
+	130.0       alice           8/1  14:56 Transfer input files failure at ⋯
 
-	$ condor_q -better-analyze JOB-ID -pool POOL-NAME
+In this particular case, a user had this in his or her HTCondor submit file:
```

Comment:
```suggestion
In this particular case, the message indicates that HTCondor cannot transfer back output files, likely because they were never created. This hold condition can be triggered if a user has this option in their HTCondor submit file:
```